R/utils-shared.R

```diff
@@ -448,13 +448,31 @@ getWaterDataUrl <- function(data) {
 #' @param data A list of gsplot objects to display on the plot.
 #' @return gsplot object with approval bar rectangle styles applied.
 ApplyApprovalBarStyles <- function(object, data) {
+  ylim <- ylim(object)$side.2
+  ylog <- object$global$par$ylog
+  
+  if (is.null(ylog)) {
+    # presume the semantics of NULL as FALSE, which may or not be correct, but 
+    # prevents the code from terminating here
+    ylog <- FALSE
+  }
```

Comment:
remove this if structure and include a call to isEmptyOrBlank in L466 (see comment below)